### PR TITLE
chore: create watch directory if not exist

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/serverBuilder.ts [hunk 1]
```diff
@@ -49,4 +52,20 @@ export class ServerBuilder {
   private registerPostRoutesMiddleware(): void {
     this.serverInstance.use(getErrorHandlerMiddleware((message) => this.logger.log('error', message)));
   }
+
+  private createWatchDirectory(): void {
+    try {
+      const mountDir = this.config.get<string>('mountDir');
```
Comment (asafMasa) [inline src/serverBuilder.ts:58]: move the watch dir to  containerConfig "registerExternalValues" method - and use it in both places (file mapper and this one) - to have the same logic called once

File: src/serverBuilder.ts [hunk 2]
```diff
@@ -49,4 +52,20 @@ export class ServerBuilder {
...
+      const watchDir = this.config.get<string>('watcher.watchDirectory');
+      const relativeWatchDirPath = join(mountDir, watchDir);
+      if (!existsSync(relativeWatchDirPath)) {
+        this.logger.log('info', `watch directory: '${watchDir}' is not exists, creating in path: ${relativeWatchDirPath}.`);
+        mkdirSync(relativeWatchDirPath);
+      }
+    } catch (error) {
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      this.logger.log('error', `error occured while creating watch directory: ${error}`);
```
Comment (asafMasa) [inline src/serverBuilder.ts:67]: add ${watchDir} to this log

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: src/serverBuilder.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/serverBuilder.ts]: fixed in 84554f6

File: src/serverBuilder.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/serverBuilder.ts]: fixed in b719e09

**Review by asafMasa — APPROVED** [review]

**Pull request merged**